JL/GUI/MainWindow.xaml.cs

```diff
@@ -388,6 +394,15 @@ private void MainTextBox_PreviewMouseLeftButtonUp(object sender, MouseButtonEven
 
             FirstPopupWindow.LookupOnSelect(MainTextBox);
         }
+        private void Button_Click(object sender, EventArgs e)
```

Comment:
What was the intention behind this method? It seems unused currently.

Reply:
Sorry, I intended to use it for the Xaml UIElement.KeyDown Event, but it's quite unreliable, so I scrapped the idea but forgot to delete the function. I've committed the fix.